ts/client/src/client.ts

```diff
@@ -305,9 +305,9 @@ export class OpenBookV2Client {
     return market.publicKey;
   }
 
-  public findOpenOrdersIndexer(): PublicKey {
+  public findOpenOrdersIndexer(owner: PublicKey): PublicKey {
```

Comment:
Can you do here the same as in the other function `owner: PublicKey = this.walletPk`

Reply:
sure, done

Reply:
apologies for the commit showing up before the review - I spoof my commit times to always be 00:00:00 UTC